io.github.hakuneko.HakuNeko.metainfo.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>io.github.hakuneko.HakuNeko</id>
+	<launchable>io.github.hakuneko.HakuNeko.desktop</launchable>>
+	<name>HakuNeko<name>
+	<project_license>Unlicense</project_license>
+	<developer_name>Ronny Wegener</developer_name>
+  	<summary>Manga & Anime Downloader for Linux, Windows & MacOS</summary>
+  	<metadata_license>CC0-1.0</metadata_license>
+	<url type="homepage">https://github.com/manga-download/hakuneko</url>
+	<url type="bugtracker">https://github.com/manga-download/hakuneko/issues</url>
+	<url type="help">https://github.com/manga-download/hakuneko/wiki</url>
+	<description>
+		<p>HakuNeko is a cross-platform downloader for manga and anime from various websites. 
+		HakuNeko was made to help users downloading media for circumstances that require offline usage. 
+		The philosophy is ad-hoc consumption, get it when you going to read/watch it. 
+		It is not meant to be a mass downloader to stock up thousands of chapters that are just collected and will probably never be read.
+	</description>
+	<screenshots>
+   	 <screenshot type="default">
+      <image>https://raw.githubusercontent.com/manga-download/hakuneko/master/assets/screenshot-pages.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://raw.githubusercontent.com/manga-download/hakuneko/master/assets/screenshot-video.png</image>
+    </screenshot>
+	<screenshot>
+      <image>https://raw.githubusercontent.com/manga-download/hakuneko/master/assets/screenshot-reader.png</image>
+    </screenshot>
+	<screenshot>
+      <image>https://raw.githubusercontent.com/manga-download/hakuneko/master/assets/screenshot-websites.png</image>
+    </screenshot>
+	<screenshot>
+      <image>https://raw.githubusercontent.com/manga-download/hakuneko/master/assets/screenshot-settings.png</image>
+    </screenshot>
+  	</screenshots>
+	<releases>
+    	<release version="v6.1.7" date="2020-01-18"/>
+	</releases>
+	<content_rating type="oars-1.0">
+		<content_attribute id="drugs-alcohol">intense</content_attribute>
```

Comment:
Not sure if that's the right OARS as it depends on the content you watch on it?

Reply:
The OARS rating is the author's rating (@ronny1982) of the program.

Reply:
The software provides access to thousands of manga, webtoons, hentai, ...
They contain mixed genres of all types.

Reply:
I think it applies to content shipped *by* the app, not necessarily when it provides access to external resources.

Reply:
So what would be the best course of action about the OARS rating?

Reply:
Let's leave it as is.